File: exercise/examples/factorial_iteration.c
#include <stdio.h>
int factorial(int );
int main (void)
{
	int number ;
	printf ("Enter the number :");
	scanf ("%d",&number);
	printf ("%d\n",factorial(number));
	return 0;

}
int factorial (int number)
{
	int result = 1;
	while (number > 0)
	{
		result *= number;
		number --;
		
	}
	return result ;


}

File: exercise/factorial_iteration.py
print ("""******************

Faktorial computing program 

To quit,please enter q...

*******************""")
#Prompt one number from the user .If user press q ,quit the program . We expect program to run until user press q ...

while (True):
    carpim=1
    sayi=raw_input ("Please enter a number:")
    if (sayi=="q"):
        print ("Exiting program...")
        break
    else :
        sayi=int (sayi)
        while (sayi>0):
            carpim*=sayi
            sayi-=1
    print ("Factorial : "+ str (carpim) )

    
